index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcd34e45ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="flex-container outer-container">
+        <header id="navbar" class="flex-container">
+            <!--- Item to display logo -->
+            <div class="flex-item" id="logo">
+                Welcome
+            </div>
+            <div class="flex-item remove-nav-item">
+                <a href="#">Portfolio</a>
+            </div>
+            <div class="flex-item remove-nav-item">
+                <a href="#">About</a>
+            </div>
+            <div class="flex-item remove-nav-item">
+                <a href="#">Contact</a>
+            </div>
+            <div class="flex-item" id="link">
+                <i class="fa-solid fa-bars"></i></i>
+            </div>
+        </header>
+        <section class="flex-container main-container">
+                <div><h1 class="intro">Hey There. I'm Ahmad.</h1></div>
+                <div></div><h1 class="profession">I'm a Full-stack Web Developer.</h1></div>
+                <div><p>I can help you build interactive and responsive websites. To be sure about the quality of my work, you may view my portfolio.</p></div>
+        </section>
+        <section class="lower-container social-links">
+            <ul class="icons">
+                <li><a href=#"><i class="fa-brands fa-github"></i></a></li>
+                <li><a href=#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                <li><a href=#"><i class="fa-regular fa-hand-peace"></i></a></li>
+                <li><a href=#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href=#"><i class="fa-sharp fa-solid fa-m"></i></a></li>
+            </ul>          
+        </section>
+    </section>
+    <section class="down-arrow">
+        <i class="fa-solid fa-chevron-down"></i>
+    </section>
```

Comment:
- [ ] Kindly consider taking a `div` tag here instead of `section` because we may use section tag for bigger parts of our HTML page for the best semantics.